### ISSUE TITLE
DeviceInfo.Model not pulling back correct value.

### ISSUE THREAD

**Issue by Celebane** [DarthAffe/CUE.NET#24]
I wrote the below C# using CUE.NET 0.1.0-CI00011.  Notice that the model came back as Chinese characters rather than the expected "K95 RGB".  I then wrote a quick c++ console app with the below lines and the Model is successfully return as well as the correct enumerations for the other values.

**C#**

``` c#
CorsairKeyboard keyboard = CueSDK.KeyboardSDK;
if (keyboard == null)
    throw new WrapperException("No Keyboard Found!");

txtbxModel.Text = keyboard.KeyboardDeviceInfo.Model.ToString();
txtbxType.Text = keyboard.KeyboardDeviceInfo.Type.ToString();
txtbxPhysLayout.Text = keyboard.KeyboardDeviceInfo.PhysicalLayout.ToString();
txtbxLogicalLayout.Text = keyboard.KeyboardDeviceInfo.LogicalLayout.ToString();
txtbxCapabilites.Text = keyboard.KeyboardDeviceInfo.CapsMask.ToString();
```

![image](https://cloud.githubusercontent.com/assets/15112037/10531255/8b3adbfe-7364-11e5-87cd-8c98b2800d22.png)

**C++**

``` c++
// I only have 1 device so just passing devicecount -1 (essentially 0)/
const auto KBInfo = CorsairGetDeviceInfo(CorsairGetDeviceCount() -1);
std::cout << (*KBInfo).model << std::endl;
std::cout << (*KBInfo).logicalLayout << std::endl;
std::cout << (*KBInfo).physicalLayout << std::endl;
std::cout << (*KBInfo).capsMask << std::endl;
std::cout << (*KBInfo).type << std::endl;
```

![image](https://cloud.githubusercontent.com/assets/15112037/10531295/0f0588da-7365-11e5-92cb-62a568c5a558.png)

**Comment by DarthAffe** [comment]
Problem should be fixed now (v0.1.0-CI00012). Feel free to reopen this issue if it occurs again.

**Comment by Celebane** [comment]
All good now.. thanks for looking at it so quickly.
